carbon-design-system/carbon
# fix(numberinput): correct stepper behavior, typecast internal value to number

Closes #12165

#### Changelog

**Changed**

- use `parseInt` to ensure internal value is always a number

#### Testing / Reviewing

- Check out the `Testing` story
  1. Enter `44` in the input via keyboard
  2. press the increment button, it should increment as expected and not go to the max value or NaN


~🛑 Don't merge until the `Testing` story is removed~
